### ISSUE TITLE
Find All References does not find references in modules within same workspace

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#74459]
- VSCode Version:  1.33.1
- OS Version: Windows 10

Steps to Reproduce:

1. `git clone https://github.com/hediet/typescript-find-references-in-workspace`
2. Open `lib/src/index.ts`, find all references to `helloWorld`
3. Open `client/src/index.ts` permanently 
4. Open `lib/src/index.ts`, find all references to `helloWorld`

Does this issue occur when all extensions are disabled?: Yes

Expected outcome:
The references found after step 2 and 4 are the same, they both find the reference in `client/src/index.ts`.

Actual outcome:
Vscode only finds the reference in `client` if any file of the `client` projected is opened. This is very unsatisfying when renaming elements.

Basically, #48474 is not fixed. I don't know whether this is a TypeScript or VSCode isssue.
It also happens if `references` are specified in `tsconfig.json`, either from `lib` to `client` or vice versa.

**Comment by hediet** [comment]
Oops. The duplicate detection actually was helpful this time.
